### PR TITLE
Multiple minor additions to FE2.0 related css

### PR DESCRIPTION
**Pull request by mko-sci**
Fixes: OX-10284

### CODE REVIEW

File: src/main/resources/default/assets/common/core.js.pasta [hunk 1]
```diff
@@ -597,3 +597,17 @@ sirius.areObjectsDeeplyEqual = function (object1, object2) {
...
 }
+
+/**@
+ * Extracts the file extension from the given filename.
+ *
+ * @param filename the filename to get the extension from
+ * @returns {string} the extension of the given filename (including the dot)
+ */
+sirius.extractFileExtension = function (filename) {
+    var a = filename.split('.');
```
Comment (sabieber) [inline src/main/resources/default/assets/common/core.js.pasta:608]: **Optional:** Please choose a better variable name. And use `const`.

File: src/main/resources/assets/design-system/message.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  // This is no shadow, but an auxiliary border.
+  box-shadow: 0 0 1px $sirius-black;
+}
+
+.sci-message-banner-success {
+  @extend .sci-message-banner;
+  border-left: 0.25rem solid $sirius-deep-blue-dark;
+}
+
+.sci-message-banner-error{
```
Comment (sabieber) [inline src/main/resources/assets/design-system/message.scss:15]: **Optional:** File is not auto-formatted. Please set auto-formatting as save-action in IntelliJ

File: src/main/resources/assets/design-system/form.scss [hunk 3]
```diff
@@ -30,3 +30,16 @@
...
+
+// DISABLED STATE
+// --------------
+// Disabled and read-only inputs
+input[disabled],
+select[disabled],
+textarea[disabled],
+input[readonly],
+select[readonly],
+textarea[readonly] {
```
Comment (sabieber) [inline src/main/resources/assets/design-system/form.scss:42]: **Required:** Those are to generic and would also apply to non-scireum fields in websites where our code is loaded. please scope them with a sci class. Like `input.sci-text-input[disabled]` eg.

**Review by sabieber — CHANGES_REQUESTED** [review]

**Review by idlira — APPROVED** [review]

File: src/main/resources/default/assets/common/core.js.pasta [hunk 4]
```diff
@@ -597,3 +597,17 @@ sirius.areObjectsDeeplyEqual = function (object1, object2) {
...
+ *
+ * @param filename the filename to get the extension from
+ * @returns {string} the extension of the given filename (including the dot)
+ */
+sirius.extractFileExtension = function (filename) {
+    const array = filename.split('.');
+    if (array.length === 1 || (array[0] === '' && array.length === 2)) {
+        return '';
+    }
+    return '.' + array.pop();
```
Comment (sabieber) [inline src/main/resources/default/assets/common/core.js.pasta:612]:
```suggestion
    const nameAndExtension = filename.split('.');
    if (nameAndExtension.length === 1 || (nameAndExtension[0] === '' && nameAndExtension.length === 2)) {
        return '';
    }
    return '.' + nameAndExtension.pop();
```

**Review by sabieber — APPROVED** [review]

**Pull request merged**